dex/networks/eth/txdata_test.go

```diff
@@ -18,16 +22,60 @@ func mustParseHex(s string) []byte {
 	return b
 }
 
+func initiationsAreEqual(i1, i2 ETHSwapInitiation) bool {
+	return i1.RefundTimestamp.Cmp(i2.RefundTimestamp) == 0 &&
+		bytes.Equal(i1.SecretHash[:], i2.SecretHash[:]) &&
+		bytes.Equal(i1.Participant.Bytes(), i2.Participant.Bytes()) &&
+		i1.Value.Cmp(i2.Value) == 0
+}
+
 func TestParseInitiateData(t *testing.T) {
-	participantAddr := common.HexToAddress("8d83B207674bfd53B418a6E47DA148F5bFeCc652")
-	secretHashSlice := mustParseHex("4aec4dc47fc6bd1fd5091c1aa4067c7fbd6bbcdb476209756354ec784d6082dc")
+	participantAddr := common.HexToAddress("345853e21b1d475582E71cC269124eD5e2dD3422")
 	var secretHash [32]byte
-	copy(secretHash[:], secretHashSlice)
+	var secretHash2 [32]byte
```

Comment:
Again sorry for the nit pik but would prefer secretHashA and secretHashB, or something like that rather than a number. Feel free to ignore.

Reply:
It's all good, go ahead and nit pik :). I'll update it. I think it's nicer too.